### PR TITLE
Rename to gewia

### PR DESCRIPTION
**Pull request by msentwickler**
## Related Issue
#12 

## Type of change

- [x] Bug fix
- [ ] Feature request
- [ ] Documenting code

## Checklists

- [x] I referenced the according Issue
  - [x] I created an Issue
  - [x] I put a link in the Section [Related Issue](#related-issue)
- [ ] I tested my code with multiple browsers
- [x] My commit messages are good and understandable

### CODE REVIEW

File: .storybook/manager.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import {addons} from '@storybook/addons';
 
-import eeduTheme from './e-eduTheme'
+import eeduTheme from './gewiaTheme'
 
 addons.setConfig({
     theme: eeduTheme,
```
Comment (Akida31) [inline .storybook/manager.js:6]: rename this

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# E-Edu Frontend
+# Gewia Frontend
 
-![Build](https://github.com/E-Edu/frontend/workflows/Build/badge.svg?style=flat&logo=appveyor)
-![GitHub](https://img.shields.io/github/license/E-Edu/frontend)
-[![GitHub contributors](https://img.shields.io/github/contributors/E-Edu/frontend.svg?style=flat)](https://GitHub.com/E-Edu/frontend/graphs/contributors/)
+![Build](https://github.com/Gewia/frontend/workflows/Build/badge.svg?style=flat&logo=appveyor)
+![GitHub](https://img.shields.io/github/license/Gewia/frontend)
+[![GitHub contributors](https://img.shields.io/github/contributors/Gewia/frontend.svg?style=flat)](https://GitHub.com/Gewia/frontend/graphs/contributors/)
```
Comment (Akida31) [inline README.md:5]: gewia instead of Gewia

File: README.md [hunk 3]
```diff
@@ -24,13 +24,16 @@ Start storybook server:
 
 You want to contribute to this project? Great!
 
-[Git Projekt tasks](https://github.com/E-Edu/frontend/projects/2)<br/>
-[Frontend Convention](https://github.com/E-Edu/frontend/blob/experimental/contribution.md)<br/>
+[Git Projekt tasks](https://github.com/Gewia/frontend/projects/2)<br/>
+[Frontend Convention](https://github.com/Gewia/frontend/blob/experimental/contribution.md)<br/>
```
Comment (Akida31) [inline README.md:28]: gewia instead of Gewia

File: src/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -23,7 +23,7 @@ class Footer extends React.Component<FooterProps> {
                     <Link to="/credits" className="credits-link">
                         {t.t('component.footer.credits', 'Credits')}
                     </Link>
-                    <a href="https://github.com/E-Edu/general" className="github-link">
+                    <a href="https://github.com/Gewia/general" className="github-link">
```
[the remark this answers is not in the document]
Comment (Akida31) [reply src/components/Footer/Footer.tsx:26]: gewia instead of Gewia

File: src/i18n/de_DE.json [hunk 5]
```diff
@@ -47,7 +47,7 @@
         },
         "landingPage": {
             "title": "Education for <span>You</span>",
-            "description": "E-Edu ist eine Lernplattform die im Rahmen des Hackathon #WirVsVirus vom Team  <span>\"The Morpheus Tutorials\"</span> erstellt wurde.<br>E-Edu bietet eine Lernplattform für Schüler die von Lehrern erstellte Aufgaben bearbeiten können."
+            "description": "Gewia ist eine Lernplattform die im Rahmen des Hackathon #WirVsVirus vom Team  <span>\"The Morpheus Tutorials\"</span> erstellt wurde.<br>Gewia bietet eine Lernplattform für Schüler die von Lehrern erstellte Aufgaben bearbeiten können."
         },
         "taskList": {
             "questions": "Fragen",
```
Comment (Akida31) [inline src/i18n/de_DE.json:53]: can this file be deleted?

File: src/i18n/en_US.json [hunk 6]
```diff
@@ -47,7 +47,7 @@
         },
```
Comment (Akida31) [inline src/i18n/en_US.json:47]: delete this file?

**Comment by Akida31** [comment]
you forgot these:
docker-compose.yml -> rename image
config.json -> rename apis

**Comment by Akida31** [comment]
should Gewia be capitalized or not?
Gewia vs. gewia

File: src/components/Footer/Footer.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (msentwickler) [reply src/components/Footer/Footer.tsx:26]: It's writen capital.

File: README.md (hunk 2 above)
Comment (msentwickler) [reply README.md:5]: same

File: README.md (hunk 3 above)
Comment (msentwickler) [reply README.md:28]: same

File: src/i18n/de_DE.json (hunk 5 above)
Comment (msentwickler) [reply src/i18n/de_DE.json:53]: maybe, i'll test it

File: src/i18n/en_US.json (hunk 6 above)
Comment (msentwickler) [reply src/i18n/en_US.json:47]: same

**Pull request merged**